config/locales/zh.yml

```diff
@@ -1820,13 +1820,13 @@ user_mailer.new_device_sign_in_after_2fa.info_p2: If you recognize this activity
 user_mailer.new_device_sign_in_after_2fa.info_p3_html: If this wasn’t you, %{reset_password_link_html} and change your %{authentication_methods_link_html} immediately.
 user_mailer.new_device_sign_in_after_2fa.reset_password: reset your password
 user_mailer.new_device_sign_in_after_2fa.subject: New sign-in and authentication with your %{app_name} account
-user_mailer.new_device_sign_in_attempts.events.sign_in_after_2fa: Authenticated
+user_mailer.new_device_sign_in_attempts.events.sign_in_after_2fa: 身份已证实
 user_mailer.new_device_sign_in_attempts.events.sign_in_before_2fa: Signed in with password
-user_mailer.new_device_sign_in_attempts.events.sign_in_unsuccessful_2fa: Failed to authenticate
-user_mailer.new_device_sign_in_attempts.new_sign_in_from: New sign-in potentially located in %{location}
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.one: Your %{app_name} email and password were used to sign in from a new device but <strong>failed to authenticate</strong>.
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.other: Your %{app_name} email and password were used to sign in from a new device but <strong>failed to authenticate %{count} times</strong>.
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.zero: Your %{app_name} email and password were used to sign in from a new device but <strong>failed to authenticate</strong>.
+user_mailer.new_device_sign_in_attempts.events.sign_in_unsuccessful_2fa: 身份证实失败
+user_mailer.new_device_sign_in_attempts.new_sign_in_from: 新登录可能是在 %{location}
+user_mailer.new_device_sign_in_before_2fa.info_p1_html.one: Your %{app_name} email and password were used to sign in from a new device but <strong>身份证实失败</strong>.
+user_mailer.new_device_sign_in_before_2fa.info_p1_html.other: Your %{app_name} email and password were used to sign in from a new device but <strong>身份证实失败%{count}次</strong>.
```

Comment:
Per the [Slack conversation](https://gsa-tts.slack.com/archives/C01710KMYUB/p1716228624508659), I need the complete text before making the PR ready for review.